### PR TITLE
Draft: 45 disabling resizing

### PR DESCRIPTION
**Pull request by tommccaughey**

### CODE REVIEW

File: src/MensaWidget/MWFrame.class.st [hunk 1]
```diff
@@ -65,6 +65,29 @@ MWFrame class >> scrollBarThickness [
 	^ 20
 ]
 
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'tmc 6/10/2024 18:13'
+}
+MWFrame >> addExpandBox [
+
+	^nil
```
Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:74]: space

File: src/MensaWidget/MWFrame.class.st [hunk 2]
```diff
@@ -65,6 +65,29 @@ MWFrame class >> scrollBarThickness [
...
+
+]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'tmc 6/10/2024 18:12'
+}
+MWFrame >> addGrips [
+
+	^nil
```
Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:86]: space

File: src/MensaWidget/MWFrame.class.st [hunk 3]
```diff
@@ -82,6 +105,18 @@ MWFrame >> buildFrameContent [
 	self addPaneMorph: contentMorph
 ]
 
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'tmc 6/10/2024 18:12'
+}
+MWFrame >> expandToFullScreen [
+
+	^nil
```
Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:114]: space

File: src/MensaWidget/MWFrame.class.st [hunk 4]
```diff
@@ -111,7 +146,7 @@ MWFrame >> initialize [
 		buildFrameContent;
 		openInWorld;
 		extent: self class frameSize;
-		removeMaximize;
+		"removeMaximize;"
 		removeScalability
```
Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:150]: you can remove both lines

File: src/MensaWidget/MWFrame.class.st
Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:1]: for all the overridden methodes give them a category like scalability

Comment (lchristus) [inline src/MensaWidget/MWFrame.class.st:1]: remove the old methodes if they are not used anymore

**Review by lchristus — CHANGES_REQUESTED** [review]
Very nice implementation that is clean. Just implement the slight feedback

File: src/MensaWidget/MWFrame.class.st (hunk 4 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrame.class.st:150]: done

File: src/MensaWidget/MWFrame.class.st (hunk 3 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrame.class.st:114]: done

**Review by lchristus — CHANGES_REQUESTED** [review]
Now we only require the remove halo extent and also we should think about if we can write test for that by eg. fullscreening it like the doubleclick would and check that the extent stays unchanged

File: src/MensaWidget/MWFrameTest.class.st [hunk 5]
```diff
@@ -81,6 +81,70 @@ MWFrameTest >> testCorrectTitle [
...
+	#category : #tests,
+	#'squeak_changestamp' : 'tmc 6/11/2024 18:17'
+}
+MWFrameTest >> testForButton [
+
+	| bool |
+	
+	bool := true.
+	self widgetFrame submorphsDo: [:each |
+		each submorphsDo: [:each2 |
```
Comment (lchristus) [inline src/MensaWidget/MWFrameTest.class.st:94]: instead you can use allMorphs

File: src/MensaWidget/MWFrameTest.class.st [hunk 6]
```diff
@@ -81,6 +81,70 @@ MWFrameTest >> testCorrectTitle [
...
+			(each2 respondsTo: #actionSelector) ifTrue: [
+				(each2 actionSelector = #expandBoxHit) ifTrue: [bool:= false]]]].
```
Comment (lchristus) [inline src/MensaWidget/MWFrameTest.class.st:96]: you can have this as one and statement

File: src/MensaWidget/MWFrameTest.class.st [hunk 7]
```diff
@@ -81,6 +81,70 @@ MWFrameTest >> testCorrectTitle [
...
+MWFrameTest >> testFullScreen [
+	
+	| bool size | 
+	
+	size := 1 @ 1.
+	bool := false.
+	self widgetFrame extent: size.
+	self widgetFrame expandToFullScreen.
+	(self widgetFrame extent = size) ifTrue: [bool := true].
+	self assert: bool
```
Comment (lchristus) [inline src/MensaWidget/MWFrameTest.class.st:114]: kannst hier direkt self asset: (self widgetFrame extent = size) und würde das auch nochmal vor das expand to fullscreen machen, damit du sichergehst das das geklappt hat

File: src/MensaWidget/MWFrameTest.class.st [hunk 8]
```diff
@@ -81,6 +81,70 @@ MWFrameTest >> testCorrectTitle [
...
+	#'squeak_changestamp' : 'tmc 6/11/2024 17:07'
+}
+MWFrameTest >> testGripMorphs [
+
+	| bool |
+	
+	bool := true.
+	self widgetFrame submorphsDo: [:each | 
+		(each class  inheritsFrom:CornerGripMorph) ifTrue:
+		[bool := false]].
```
Comment (lchristus) [inline src/MensaWidget/MWFrameTest.class.st:128]: tab

File: src/MensaWidget/MWFrameTest.class.st [hunk 9]
```diff
@@ -81,6 +81,70 @@ MWFrameTest >> testCorrectTitle [
...
+MWFrameTest >> testHalo [
+	
+	| bool size | 
+	
+	size := 1 @ 1.
+	bool := false.
+	self widgetFrame extent: size.
+	self widgetFrame setExtentFromHalo: 5 @ 5.
+	(self widgetFrame extent = size) ifTrue: [bool := true].
+	self assert: bool
```
Comment (lchristus) [inline src/MensaWidget/MWFrameTest.class.st:145]: same wie beim fullscreen test

File: src/MensaWidget/MWFrameTest.class.st (hunk 5 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:94]: Done

File: src/MensaWidget/MWFrameTest.class.st (hunk 6 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:96]: Done

File: src/MensaWidget/MWFrameTest.class.st (hunk 7 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:114]: done

File: src/MensaWidget/MWFrameTest.class.st (hunk 9 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:145]: done

File: src/MensaWidget/MWFrameTest.class.st (hunk 8 above)
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:128]: done

File: src/MensaWidget/MWFrameTest.class.st [hunk 10]
```diff
@@ -81,6 +81,72 @@ MWFrameTest >> testCorrectTitle [
...
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'tmc 6/12/2024 12:03'
+}
+MWFrameTest >> testHalo [
+	
+	| size | 
+	
+	size := 1 @ 1.
+
```
[the remark this answers is not in the document]
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:142]: done

File: src/MensaWidget/MWFrameTest.class.st [hunk 11]
```diff
@@ -81,6 +81,72 @@ MWFrameTest >> testCorrectTitle [
...
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'tmc 6/12/2024 12:01'
+}
+MWFrameTest >> testFullScreen [
+	
+	| size | 
+	
+	size := 1 @ 1.
+
```
[the remark this answers is not in the document]
Comment (tommccaughey) [reply src/MensaWidget/MWFrameTest.class.st:110]: done

**Review by lchristus — APPROVED** [review]

**Comment by lchristus** [comment]
Eventhough this is a very nice implementation to tackle the issue the requirements have changed and this will not be required anymore

**Pull request closed without merging**